packages/data-service/src/run-command.ts

```diff
@@ -193,6 +193,7 @@ export type CollectionInfoNameOnly = {
 export type CollectionInfo = CollectionInfoNameOnly & {
   /** @see https://docs.mongodb.com/manual/reference/method/db.createCollection/#mongodb-method-db.createCollection */
   options: Document;
+  name: string;
```

Comment:
With a risk to embarrass myself with my poor typescript knowledge, why do we need this here? 😅 My expectation is that since we're merging this with the `CollectionInfoNameOnly`, we already have a `name: string` field.

Reply:
Oh duh. Yeah, just missed that this was already part of the type, that's all 🙈